spamc.hsl

```diff
@@ -48,12 +48,23 @@ function spamc($fp, $opts = [])
 	if (!$result) return ["error" => "No scan result"];
 	if (!pcre_match("EX_OK", $result)) return ["error" => "Bad scan result"];
 
-	$result = pcre_match(''Spam: (.*) ; (.*) / (.*)\r\n\r\n(.*)'', $result);
+	$spam_result = pcre_match(''Spam: (.*) ; (.*) / (.*)'', $result);
+	$spam_report = pcre_match(''/--\n([ \-\d]\d.*\n)+/s'', $result);
+	$spam_report = str_strip($spam_report[1]);
+
+	$symbol_scores = [];
```

Comment:
something like this would get the patterns in one regex

```
$symbol_scores = [];
if ($match = pcre_match_all(#/^\s*(-?\d+(?:\.\d+)?) ([^\s]+)/m, $spam_report))
  $symbol_scores = array_map(number, array_combine(...array_reverse($match[1:])));
```